android/IronSourcePlugin.java

```diff
@@ -0,0 +1,500 @@
+package com.tealeaf.plugin.plugins;
```

Comment:
looks like this file has been moved from elsewhere. Either we should remove this or it's copy.

Reply:
Everything is fine here